Asterix/Example_param_file.ini

```diff
@@ -230,6 +230,17 @@ onbench=True
     # If monochromatic images (nb_wav = 1 or Delta_wav = 0), all these options are equivalent
     # Not case sensitive
     polychromatic = 'centralwl'
+
+    # if polychromatic = 'multiwl'
+    # Number of monochromatic images for the correction. Must be an odd integer,
```

Comment:
This should be "Number of monochromatic images for the *estimation*.", not the correction if I understand your PR description correctly, since the correction and simulation parameters are now the same and only the estimation parameters can differ, with certain requirements.

Reply:
you are right. In practice its both. But since for now Correction is only synthetic (we do correction with asterix at watever wl we like) compared to the estimation which is "in the real world" (it correspond to specific filters on the testbed), I decided that it make more sense to call it estimation : if we change the estimation wl, we can easily change the correction wl accordingly.